crates/core/src/prelude/README.md

```diff
@@ -0,0 +1,21 @@
+# extism js-pdk types
+
+TypeScript definitions for the Extism [JS Plugin Development Kit](https://github.com/extism/js-pdk).
+
+To install these types, add them to your `tsconfig.json`'s `typeRoots`:
```

Comment:
While this works, it's not ideal as it doesn't set the appropriate default library, nor does it override the default of including all type packages in these folders, so you may end up with node, jest, etc globals.

I updated the root readme under the bundler section, with the usage I personally suggest.

Reply:
Ah, good catch, thank you!